### PR TITLE
Fidget spinner #4119

### LINKED ISSUE #4119: [New game]: Fidget Spinner
### 🎮 Game Request

Game logic and basic description

The logic of the game is that the turns will keep increasing as you press the space bar, and it will reduce its speed and stop at a point where you stop pressing the space bar

### Point down the features

Game points

We use tkinter for this

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by nisargaa20**
## PR Description 📜

Please include summary related to the issue you have fixed and describe your PR in brief over here by specifying the issue number on which you were working below
Fixes # <your_issue_number>

Fidget Spinner is an interactive online simulation of a physical fidget spinner. This digital version allows users to experience the satisfying and stress-relieving sensation of spinning a fidget spinner directly from their browser.

Interactive Spinning: Users can spin the fidget spinner by dragging the mouse across the screen.
Speed Display: The current speed of the spinner is displayed, giving users real-time feedback.
Custom Speed Setting: Users can set a custom speed for the fidget spinner, allowing for a personalized experience.
Automatic Spin: For those who prefer not to drag their mouse, a button is available to automatically spin the fidget spinner.
How to Play? 🕹️
Drag to Spin: Click and drag the mouse across the spinner to make it spin. The faster you drag, the faster the spinner will rotate.
View Speed: Watch the speedometer to see how fast the spinner is going.
Set Custom Speed: Use the input box to set a custom speed for the spinner.
Automatic Spin: Click the "Spin" button to make the spinner spin without dragging.

Fixed #4119

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [ ] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

--- 
<br>

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @nisargaa20 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Comment by nisargaa20** [comment]
@kunjgit Kindly review it sooner as it getting conflicts

**Comment by nisargaa20** [comment]
@kunjgit kindly add labels

**Comment by nisargaa20** [comment]
@kunjgit kindly add labels and review it . It has been 4 days I need to raise new issue

**Comment by nisargaa20** [comment]
@ishita-43 Kindly check it out the PR and add labels

**Comment by nisargaa20** [comment]
@kunjgit Its been a week kindly check it out ( conflicting is occuring)

File: README.md [hunk 1]
```diff
@@ -317,12 +317,22 @@ This repository also provides one such platforms where contributers come over an
...
+
+[Fidget Spinner Game](https://github.com/kunjgit/GameZone/tree/main/Games/Fidget_Spinner_Game)|
 | [Color The Page](https://github.com/kunjgit/GameZone/tree/main/Games/Color_The_Page)|
 |[Building Blocks Game](https://github.com/kunjgit/GameZone/tree/main/Games/Building_Block_Game)|
 |[Cartoon character guessing game](https://github.com/kunjgit/GameZone/tree/main/Games/Cartoon_Character_Guessing_Game)|
+
 |[Carrom Board Game](https://github.com/kunjgit/GameZone/tree/main/Games/carrom)|
 | [Number_Recall_Game](https://github.com/kunjgit/GameZone/tree/main/Games/Number_Recall_Game)               |
+
+| [Hit_the_hamster] (https://github.com/kunjgit/GameZone/tree/main/Games/Hit_the_hamster)                |
```
Comment (ishita-43) [inline README.md:331]: delete this duplicate entry from here

File: assets/js/gamesData.json [hunk 2]
```diff
@@ -2099,9 +2099,37 @@
...
+
+
+  "gameTitle": "rapid_click_frenzy",
+  "gameUrl": "rapid_click_frenzy",
+  "thumbnailUrl": "rapidgame.png"
+
+ 
+  
+}
+  ,"417":{
```
Comment (ishita-43) [inline assets/js/gamesData.json:2111]: remove all these entry and place your below each entry with no 417
or arrange the entry in the order 
417
409
418
410
and so on at the bottom of file

**Review by ishita-43 — CHANGES_REQUESTED** [review]
Make the desired changes!!

**Comment by nisargaa20** [comment]
@ishita-43 done check it out

File: assets/js/gamesData.json [hunk 3]
```diff
@@ -2112,43 +2112,43 @@
     "gameTitle": "Pen_Pointer_Fight",
   "gameUrl": "PenPointerFight",
   "thumbnailUrl": "PenPointerFight.png"
-},"418":{
+},"409":{
```
Comment (ishita-43) [inline assets/js/gamesData.json:2115]: change as 418

File: assets/js/gamesData.json [hunk 4]
```diff
@@ -2112,43 +2112,43 @@
...
   "gameTitle": "Fidget_Spinner_Game",
   "gameUrl": "Fidget_Spinner_Game",
   "thumbnailUrl": "Fidget.png"
-
-
+},
+"418":{
```
Comment (ishita-43) [inline assets/js/gamesData.json:2120]: 409

File: assets/js/gamesData.json [hunk 5]
```diff
@@ -2112,43 +2112,43 @@
...
   "gameTitle": "Brick Buster",
    "gameUrl": "Brick Buster",
    "thumbnailUrl": "Brick.png"
  }
 
 
-  }
-},"408":{
+  
+},"417":{
```
Comment (ishita-43) [inline assets/js/gamesData.json:2134]: 410

File: assets/js/gamesData.json [hunk 6]
```diff
@@ -2112,43 +2112,43 @@
...
    "thumbnailUrl": "Brick.png"
  },
- "417":{
+ "409":{
   "gameTitle": "Soccer",
   "gameUrl": "Soccer",
   "thumbnailUrl": "Soccer"
 },
-"409":{"gameTitle": "Pen_Pointer_Fight",
+"418":{"gameTitle": "Pen_Pointer_Fight",
```
Comment (ishita-43) [inline assets/js/gamesData.json:2144]: 411

File: assets/js/gamesData.json [hunk 7]
```diff
@@ -2112,43 +2112,43 @@
...
+},
+"418":{
  "gameTitle": "Guess_The_Song",
   "gameUrl": "Guess_The_Song",
   "thumbnailUrl": "Guess_The_Song.png"
+}
 
-
-},"417":{
+},"410":{
```
Comment (ishita-43) [inline assets/js/gamesData.json:2126]: 419

File: assets/js/gamesData.json [hunk 8]
```diff
@@ -2112,43 +2112,43 @@
...
-  }
-},"408":{
+  
+},"417":{
   "gameTitle": "Brick Buster",
    "gameUrl": "Brick Buster",
    "thumbnailUrl": "Brick.png"
  },
- "417":{
+ "409":{
```
Comment (ishita-43) [inline assets/js/gamesData.json:2139]: 420

File: assets/js/gamesData.json [hunk 9]
```diff
@@ -2112,43 +2112,43 @@
...
   "gameUrl": "Soccer",
   "thumbnailUrl": "Soccer"
 },
-"409":{"gameTitle": "Pen_Pointer_Fight",
+"418":{"gameTitle": "Pen_Pointer_Fight",
 "gameUrl": "PenPointerFight",
 "thumbnailUrl": "PenPointerFight.png"
 },
-"418":{
+"410":{
```
Comment (ishita-43) [inline assets/js/gamesData.json:2148]: 421

**Review by ishita-43 — CHANGES_REQUESTED** [review]

**Comment by nisargaa20** [comment]
@ishita-43 I got confused while doing check it out if its correct

**Comment by ishita-43** [comment]
> @ishita-43 I got confused while doing check it out if its correct

ok you need to make changes in such a way that the number increase alternatingly 
eg if you ha 407 416 these two entries in the end then make increament alternatinly eg first increase first number than second number so for above examolle series would be 
407 416 than
408 417 409 418 410 and keep this increasing till all entries get nubered did you get this!!

**Comment by nisargaa20** [comment]
@ishita-43 Thank you got it now . I have made the changes check it out

**Comment by ishita-43** [comment]
Merge conflicts are arising please do one think sync your json and readme file and then add again add your game there and then upload them here!!

**Comment by nisargaa20** [comment]
@ishita-43 kindly merge it soon or else again there will be conflict if u merge others PR

**Review by ishita-43 — APPROVED** [review]
Seems good!!

**Comment by nisargaa20** [comment]
@ishita-43 Kindly merge it

**Comment by kunjgit** [comment]
Thank you  @nisargaa20 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**